media-sound/pulseaudio-daemon/pulseaudio-daemon-15.0.ebuild

```diff
@@ -148,9 +148,11 @@ DOCS=( NEWS README )
 
 S="${WORKDIR}/${MY_P}"
```

Comment:
My suggestion would be to revbump (`git mv` should be fine, I guess) just to ease debugging so we know everyone is in the same state, even if in theory it should all be the same.

Reply:
Since both media-libs/libpulse and media-sound/pulseaudio-daemon have the same change, updated both to 15.0-r1